scripts/devops_tasks/common_tasks.py

```diff
@@ -222,31 +282,98 @@ def run_check_call(
             else:
                 return err
 
+
 # This function generates code coverage parameters
 def create_code_coverage_params(parsed_args, package_name):
     coverage_args = []
     if parsed_args.disablecov:
         logging.info("Code coverage disabled as per the flag(--disablecov)")
         coverage_args.append("--no-cov")
     else:
-        current_package_name = package_name.replace('-','.')
+        current_package_name = package_name.replace("-", ".")
         coverage_args.append("--cov={}".format(current_package_name))
-        logging.info("Code coverage is enabled for package {0}, pytest arguements: {1}".format(current_package_name, coverage_args))
+        logging.info(
+            "Code coverage is enabled for package {0}, pytest arguements: {1}".format(
+                current_package_name, coverage_args
+            )
+        )
     return coverage_args
 
+
 # This function returns if error code 5 is allowed for a given package
 def is_error_code_5_allowed(target_pkg, pkg_name):
     if (
-            all(
-                map(
-                    lambda x: any(
-                        [pkg_id in x for pkg_id in MANAGEMENT_PACKAGE_IDENTIFIERS]
-                    ),
-                    [target_pkg],
-                )
+        all(
+            map(
+                lambda x: any(
+                    [pkg_id in x for pkg_id in MANAGEMENT_PACKAGE_IDENTIFIERS]
+                ),
+                [target_pkg],
             )
-            or pkg_name in NON_MANAGEMENT_CODE_5_ALLOWED
-        ):
+        )
+        or pkg_name in NON_MANAGEMENT_CODE_5_ALLOWED
+    ):
         return True
     else:
-        return False
\ No newline at end of file
+        return False
+
+
+# This function parses requirement and return package name and specifier
+def parse_require(req):
+    req_object = Requirement.parse(req)
+    pkg_name = req_object.key
+    spec = SpecifierSet(str(req_object).replace(pkg_name, ""))
+    return [pkg_name, spec]
+
+
+# This method installs package from a pre-built whl
+def install_package_from_whl(
```

Comment:
I believe we've already discussed and have a follow-up item to unify this with the one in `create_and_install_wheel` yes?